src/UXClient/Models/Strings.ts

```diff
@@ -77,9 +77,12 @@ class Strings {
         "timezone selection": "timezone selection",
         "Start time input": "Start time input",
         "End time input": "End time input",
+        "*": "*",
         "snap end time to latest": "snap end time to latest",
         "zoom in": "zoom in",
         "zoom out": "zoom out",
```

Comment:
Do you still need these 2?

Reply:
We need this as it is used in title attribute.